proposals/299-ip-failure-count.txt

```diff
@@ -82,7 +82,7 @@ Ticket: https://trac.torproject.org/projects/tor/ticket/27491
    These values will be stored as 32-bit unsigned integers.
```

Comment:
This line should also say "16-bit integers".

Reply:
OK.